multiqc/utils/config.py

```diff
@@ -219,6 +219,10 @@ def mqc_load_userconfig(paths=()):
     # Load and parse installation config file if we find it
     mqc_load_config(os.path.join(os.path.dirname(MULTIQC_DIR), "multiqc_config.yaml"))
 
+    # Load and parse a config file in $XDG_CONFIG_HOME
+    # Ref: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+    mqc_load_config(os.path.join(os.environ.get("XDG_CONFIG_HOME", "~/.config"), "multiqc_config.yaml"))
```

Comment:
Not sure if we need to expand the user path in the default here?
```suggestion
    mqc_load_config(os.path.join(os.environ.get("XDG_CONFIG_HOME", os.path.expanduser("~/.config")), "multiqc_config.yaml"))
```